hexa/databases/api.py

```diff
@@ -0,0 +1,51 @@
+import psycopg2
+from django.conf import settings
+from psycopg2 import sql
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
+
+from hexa.plugins.connector_postgresql.models import Database
+
+
+def get_db_server_credentials():
+    return {
+        "role": settings.WORKSPACE_DATABASE_ROLE,
+        "password": settings.WORKSPACE_DATABASE_PASSWORD,
+        "host": settings.WORKSPACE_DATABASE_HOST,
+        "port": settings.WORKSPACE_DATABASE_PORT,
+    }
+
+
+def create_database(db_name: str):
+    credentials = get_db_server_credentials()
+
+    role = credentials["role"]
+    password = credentials["password"]
+    host = credentials["host"]
+    port = credentials["port"]
+
+    url = f"postgresql://{role}:{password}@{host}:{port}"
+    try:
+        conn = psycopg2.connect(url)
+        conn.set_isolation_level(ISOLATION_LEVEL_AUTOCOMMIT)
+        with conn.cursor() as cursor:
```

Comment:
I would be extra careful about SQL injection here. Normally we would use execute parameters but this is not possible for CREATE DATABASE operations.

Could you use the `psycopg2.sql` submodule? You can have a look here: https://www.psycopg.org/docs/sql.html ?

Reply:
Oh yes, I'll update it